Calmspace/DisplayViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  DisplayViewController.swift
+//  Calmspace
+//
+//  Created by Admin on 17/10/2023.
+//
+
+import UIKit
+import WebKit
+
+class DisplayViewController: UIViewController, WKNavigationDelegate {
+    
+    var url: URL?
+    
+    
+    @IBOutlet weak var webView: WKWebView!
+    
+    
+    @IBOutlet weak var actInt: UIActivityIndicatorView!
+    
+    @IBAction func refresh(_ sender: Any) {
+        
+        webView.reload()
+    }
+    
+    @IBAction func stop(_ sender: Any) {
+        
+        webView.stopLoading()
+    }
+    
+    @IBAction func rewind(_ sender: Any) {
+        
+        if webView.canGoBack {
+            webView.goBack() }
```

Comment:
Inconsistent braces: the closing brace needs to be on a separate line. In programming, things like these are important since it is crucial that the code is easy to read.